package.json

```diff
@@ -24,7 +24,8 @@
         "OpenShift-Local.memory": {
           "type": "number",
           "format": "memory",
-          "description": "Memory size in MiB (must be greater than or equal to '2048')"
+          "minimum": 2048,
```

Comment:
We can not have a minimum stated in the config, as that also means this is the minimum for the OpenShift preset, which is not true. Best to not mention this and let the UI handle the lower boundary

Reply:
I'm not sure I follow, because that sounds like exactly what I thought I did. :) Do you mean not mention nor enforce the minimum here and handle it elsewhere?

Reply:
we purposely removed this value from package json as this should be controlled by the extension directly: it is based on the preset. using it this way from the package.json therefore conflicts

Reply:
The removal of `memory` happened in: https://github.com/crc-org/crc-extension/commit/b43b855f29435a9315de9e4debcdabd44733401d

@evidolob WDYT?

Reply:
Yes, we remove all minimums values from package.json, as we cannot modify them on the runtime, and they are changed, based on preset. So we perform validation after user changed some preference from UI.

Reply:
Please remove this change from the PR, after which it can be approved.